src/metabase/search/ingestion.clj

```diff
@@ -47,7 +47,7 @@
       (update :archived boolean)
       (assoc
        :display_data (display-data m)
-       :legacy_input m
+       :legacy_input (dissoc m :pinned :view_count :last_viewed_at :native_query)
```

Comment:
We strip this as it was not returned by legacy search. I think it would be useful to return, but that will require updating the integration tests, which are currently agnostic.